connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -391,4 +390,158 @@ private String ownerUrl(String connector) {
 
     }
 
+    public static class ConnectorsAndTasks {
+        private final Collection<String> connectors;
+        private final Collection<ConnectorTaskId> tasks;
+        public static final ConnectorsAndTasks EMPTY =
+                new ConnectorsAndTasks(Collections.emptyList(), Collections.emptyList());
```

Comment:
Nit: seems odd to have statics after member fields.

Reply:
In general, I'm not sure our checkstyle is very opinionated w.r.t to this order. Changed to bring EMPTY first.